### PR TITLE
The Newsweek website clone

### PR DESCRIPTION
**Pull request by kunmi02**
This is the Newsweek website clone. Technologies used in this project include bootstrap classes and sass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (MuriloRoque) [inline README.md]: - [x] Remove these sections from your README if you are not using them.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1741 @@
...
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- Bootstrap CSS -->
+  <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+    integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous"> -->
+  <link rel="stylesheet" href="./style.css">
+
+  <!-- Local Boostrap
+  <link href="boo" -->
```
Comment (MuriloRoque) [inline index.html]: - [x] Please remove commented pieces of code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1741 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- Bootstrap CSS -->
```
Comment (MuriloRoque) [inline index.html:8]: - [x] You need to install Bootstrap in your project with any means mentioned in their webpage [here](https://getbootstrap.com/docs/4.5/getting-started/introduction/). Downloading the whole CSS file increases the size of your project and could impact any future changes you want to make to it. I couldn't find this suggestion on the Bootstrap page, if you have any sources instructing you to do what you did, can you please send it to me?

File: test.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```
Comment (MuriloRoque) [inline test.html]: - [x] If you are not using this file, please delete it.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes Requested 🔴 

### Hello @kunmi02 and @bashforger 👋 

### Wow, your page is magnificent, I could barely find anything to suggest, but I still found some details, please check my suggestions:

- [ ] Your page has a horizontal scroll on all page sizes, you should solve this problem as the original doesn't have it. This is bad because the user might have a bad experience if he/she needs to scroll in order to see some of the content:

Yours1

- [x] When we get near the medium size screen from the big size screen, your navigation links and the text inside of the "Debate" section break some words, kindly fix it:

Yours

- [x] On small screens, the "Featured Stories" section should come right after "Top Story", just like the original. Currently, on your page, we can see "My Turn" after "Top Story".

- [x] Please check my comments in your code below.

## Summary
- You rock, your page is nearing perfection, your attention to details is very good 👍 
- Your repository is very professional, keep up the good work 💪 
- We only need to fix these details before moving on 🚀 

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello again @kunmi02 and @bashforger :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by 0x1100010010** [comment]
Thank you very much, @MuriloRoque

**Pull request merged**